### PR TITLE
Implement move/rename the fast way.

### PR DESCRIPTION
**Pull request by maethu**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,7 +4,22 @@
 Introduction
 ============
 
-### INTRODUCTION ###
+This package implements some magical improvements compared to the default plone copy/move/rename implementation.
+
+As far as manage to get changes in to Plone/CMFCore this packages is no longer necessary.
```
Comment (jone) [inline README.rst]: something is missing in this sentence

File: README.rst [hunk 2]
```diff
@@ -4,7 +4,22 @@
...
+--------------------------
+This package provides a improved move/rename implementation.
+The more files you want to move the faster it will be compared to the default Plone implementation.
+
+Example measurement with 1 Folder, which contains 1 Document and 300 Files (almost empty PDFs):
+
+Plone: approx. 80s
+With this package: approx. 8s
+
+PR for this change is open at: XXX (Add URL to PR)
```
Comment (jone) [inline README.rst]: @maethu can you add a short summary containing _how_ the improvement is implemented?

**Comment by jone** [comment]
There is a `test_rid_is_the_same_after_moving`.
Can you also add a `test_rid_is_changed_on_copy` just to make sure that we neither did break copying nor will in the future?

File: README.rst (hunk 2 above)
Comment (maethu) [inline README.rst]: done

**Comment by maethu** [comment]
@jone cross check for copy implemented.

**Comment by jone** [comment]
👍 nice!

**Pull request merged**